### PR TITLE
Add pyMCR.

### PR DESCRIPTION
**Pull request by ericpre**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyMCR/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  license: NIST
+  license_family: PUBLIC-DOMAIN
+  license_file: LICENSE.md
+  summary: Multivariate Curve Resolution in Python
+  doc_url: https://pages.nist.gov/pyMCR
+  dev_url: https://github.com/usnistgov/pyMCR
+
+extra:
+  recipe-maintainers:
+    - CCampJr
```
Comment (ericpre) [inline recipes/pyMCR/meta.yaml]: @CCampJr, would you be happy to be a recipe maintainer for pyMCR?

File: recipes/pyMCR/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python  >=3
```
Comment (chrisburr) [inline recipes/pyMCR/meta.yaml]: Can you put one space here instead of two
```suggestion
    - python >=3
```

File: recipes/pyMCR/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - pytest-runner
+  run:
+    - numpy
+    - python  >=3
```
Comment (chrisburr) [inline recipes/pyMCR/meta.yaml]:
```suggestion
    - python >=3
```

File: recipes/pyMCR/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    - scipy
+    - scikit-learn  >=0.20
```
Comment (chrisburr) [inline recipes/pyMCR/meta.yaml]:
```suggestion
    - scikit-learn >=0.20
```

**Comment by ericpre** [comment]
Thanks @chrisburr for the quick review!

**Comment by chrisburr** [comment]
No problem, looks good so I'll merge once @CCampJr comments.

(Feel free to ping me if I miss It)

File: recipes/pyMCR/meta.yaml (hunk 1 above)
Comment (CCampJr) [reply recipes/pyMCR/meta.yaml]: I'd be happy to; though, I'm not 100% sure what that entails

Comment (chrisburr) [reply recipes/pyMCR/meta.yaml]: I means you have write access to the git repository used for the package (the "feedstock") after you accept the invite which will be automatically sent when this is merged.

Generally the update bot makes pull requests automatically and you just merge them when they pass. You should only need to manually intervene if the dependencies change and if you have issues or questions you can ping @ conda-forge/core (without the space) or ask on [Gitter](https://gitter.im/conda-forge/conda-forge.github.io).

Comment (CCampJr) [reply recipes/pyMCR/meta.yaml]: @chrisburr Thank you very much!

**Pull request merged**